projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/commandsAndMenu.tsx

```diff
@@ -35,7 +36,10 @@ export function updateVDKMenu(commands: CommandRegistry) {
           });
           const resultButtonClicked = !result.value && result.button.accept;
           if (resultButtonClicked) {
-            await jobRunRequest();
+            let response = await jobRunRequest();
+            if(response[1]){
```

Comment:
Consider destructuring this, so it's more clear. Something like

```
let { result, success } = await jobRunRequest();

if (success) {
    alert(result);
}
```

Reply:
Refactoring of the whole ui code will be introduced soon